### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
@alexmigf Correct misspellings.

No code change!

### CODE REVIEW

**Review by YordanSoares — APPROVED** [review]
✅ LGTM! Thanks for your contribution, @szepeviktor!

File: includes/documents/abstract-wcpdf-order-document.php [hunk 1]
```diff
@@ -1249,7 +1249,7 @@ public function get_number_store_year( $table_name ) {
 		if( $table_exists ) {
 			// get year for the last row
 			$year = $wpdb->get_var( "SELECT YEAR(date) FROM {$table_name} ORDER BY id DESC LIMIT 1" );
-			// default to currenty year if no results
+			// default to currently year if no results
```
Comment (dwalkerpriv) [inline includes/documents/abstract-wcpdf-order-document.php:1252]: should this be  'default to **current** year..' otherwise, all good

**Review by dwalkerpriv — APPROVED** [review]

File: includes/documents/abstract-wcpdf-order-document.php (hunk 1 above)
Comment (szepeviktor) [reply includes/documents/abstract-wcpdf-order-document.php]: Done 🍏

**Review by alexmigf — APPROVED** [review]
LGTM!

**Pull request merged**